sources/components/markdown/MarkdownView.tsx

```diff
@@ -176,6 +209,43 @@ function RenderSpans(props: { spans: MarkdownSpan[], baseStyle?: any }) {
     </>)
 }
 
+function RenderTableBlock(props: {
+    headers: string[],
+    rows: string[][],
+    first: boolean,
+    last: boolean
+}) {
+    return (
+        <View style={[style.tableContainer, props.first && style.first, props.last && style.last]}>
+            <ScrollView
+                horizontal
+                showsHorizontalScrollIndicator={true}
+                style={{ flex: 1 }}
+                contentContainerStyle={{ minWidth: '100%' }}
+            >
+                <View style={{ flexDirection: 'column', minWidth: '100%' }}>
```

Comment:
Wondering how this will look with unaligned sell widths. In any case merging this because it will surely look better than it looks now lol